docs/Server.md

```diff
@@ -616,8 +616,16 @@ Fake http injection (for testing purposes) [here](https://github.com/fastify/fas
 
 <a name="add-schema"></a>
 #### addSchema
-`fastify.addSchema(schemaObj)`, adds a shared schema to the Fastify instance. This allows you to reuse it everywhere in your application just by writing the schema id that you need.<br/>
-To learn more, see [shared schema example](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#shared-schema) in the [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md) documentation.
+`fastify.addSchema(schemaObj)`, adds a JSON schema to the Fastify instance. This allows you to reuse it everywhere in your application just by using the standard `$ref` keyword.<br/>
```

Comment:
Do we really want to stick with this feature? Given the complexity in maintaining proper JSON Schema instances with references, wouldn't it be better to relegate everything to a passed in schema container?

Reply:
I'm not sure it could works because the users that use the default validator and serializer need to add a common schema, and it would not be possible without this function.
Moreover, right now we are not managing the `$ref`s, so I think it would be better from now on

But maybe I misunderstand your idea
Could you elaborate?